app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php

```diff
@@ -5,7 +5,7 @@
  */
 
 /**
- * Multishipping checkout base abstract block
+ * Mustishipping checkout base abstract block
```

Comment:
Mustishipping?
Cc: @avoelkl @dmanners

Reply:
Hi @orlangur thanks for this. I have already reached out to @avoelkl with regards to this.

Reply:
I would rather not back/forwardort this PR as i accendentially added (and reverted later again) a commit that was supposed to be in another PR.
The typos were fixed in a seperate PR.

Reply:
@vijay-wagento @avoelkl @dmanners 
I think we should fix typos in current PR. We haven't the reason to do it in separate PR.